src/NServiceBus.RabbitMQ/RabbitMqDequeueStrategy.cs

```diff
@@ -280,7 +279,26 @@ void Purge()
             }
         }
 
+        bool MovePoisonMessage(BasicDeliverEventArgs message, string queue)
+        {
+            var connection = connectionManager.GetPublishConnection();
+            var success = true;
+
+            using (var channel = connection.CreateModel())
+            {
+                try
+                {
+                    channel.BasicPublish("", queue, false, message.BasicProperties, message.Body);
```

Comment:
In 3.x we don't actually have a way to check for `basic.return` messages, so there really isn't a point to setting it to true.